lightning/src/ln/channel.rs

```diff
@@ -2392,7 +2394,13 @@ impl<Signer: Sign> Channel<Signer> {
 	/// Handles a channel_ready message from our peer. If we've already sent our channel_ready
 	/// and the channel is now usable (and public), this may generate an announcement_signatures to
 	/// reply with.
-	pub fn channel_ready<L: Deref>(&mut self, msg: &msgs::ChannelReady, node_pk: PublicKey, genesis_block_hash: BlockHash, best_block: &BestBlock, logger: &L) -> Result<Option<msgs::AnnouncementSignatures>, ChannelError> where L::Target: Logger {
+	pub fn channel_ready<N: Deref, L: Deref>(
```

Comment:
I may have inadvertently introduced a convention, but in 1930, I always used `NS` for `NodeSigners`.